### PR TITLE
update for OTEL java

### PR DESCRIPTION
**Pull request by kubawach**
- added OTEL entries in README

### CODE REVIEW

**Review by mateuszrzeszutek — APPROVED** [review]

**Comment by kubawach** [comment]
@mmusik @bgola-signalfx @jkawecki-signalfx please have a look and merge if approved

File: README.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 
 ---
 
-<h2>Latest available versions of SignalFx Lambda wrapper layers:</h2>
+<h2>Latest available versions of Splunk Lambda wrapper layers:</h2>
 
 | Supported Languages      | Link to Supported Layer Versions |
 | ----------- | ----------- |
-| Java      | [Click for list of supported layer versions](./java/JAVA.md)     |
+| OTEL Java      | [Click for list of supported layer versions](./otel-java/OTEL-JAVA.md)     |
```
Comment (bgola-signalfx) [inline README.md:13]: I do not see the `otel-java` folder/readme in the repo.

**Review by bgola-signalfx — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kubawach) [reply README.md:13]: @bgola-signalfx looking at the scripts in https://github.com/signalfx/signalfx-lambda-layers I figured out that the first OTEL-JAVA.md along with the containing folder will be added once the layer is deployed. Correct me if I'm wrong, I can easily add "empty" md.

Comment (mmusik) [reply README.md:13]: I hope @kubawach is right ;) There is a template in signalfx-lambda-layers so if the pipeline passes it should add a new file. Let's run and correct if needed?

**Review by bgola-signalfx — APPROVED** [review]

**Pull request merged**